api/process_bundles.go

```diff
@@ -0,0 +1,465 @@
+		ErrMalformedBody.Write(w)
+		return
+	}
+
+	if len(req.Processes) == 0 {
+		ErrMalformedBody.Withf("no processes provided").Write(w)
+		return
+	}
+
+	// Get the user from the request context
+	user, ok := userFromContext(r.Context())
+	if !ok {
+		ErrUnauthorized.Write(w)
+		return
+	}
+
+	// Get the existing bundle
+	bundle, err := a.db.ProcessBundle(bundleID)
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	// Check if the user has the necessary permissions for the organization
+	if !user.HasRoleFor(bundle.OrgAddress, db.ManagerRole) && !user.HasRoleFor(bundle.OrgAddress, db.AdminRole) {
+		ErrUnauthorized.Withf("user is not admin or manager of organization").Write(w)
+		return
+	}
+
+	// Get the census for this bundle
+	census, err := a.db.Census(bundle.Census.ID.Hex())
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	// Collect all processes to add
+	var processesToAdd []internal.HexBytes
+
+	for _, processReq := range req.Processes {
+		if len(processReq) == 0 {
+			ErrMalformedBody.Withf("missing process ID").Write(w)
+			return
+		}
+		processID, err := hex.DecodeString(util.TrimHex(processReq))
+		if err != nil {
+			ErrMalformedBody.Withf("invalid process ID").Write(w)
+			return
+		}
+
+		processesToAdd = append(processesToAdd, processID)
+	}
+
+	// Find the census participants
+	orgParticipants, err := a.db.OrgParticipantsMemberships(census.OrgAddress, census.ID.Hex(), bundleIDStr)
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	// if a twofactor census add the process to the twofactor service
+	if census.Type == db.CensusTypeSMSorMail ||
+		census.Type == db.CensusTypeMail ||
+		census.Type == db.CensusTypeSMS {
+		if err := a.twofactor.AddProcess(census.Type, orgParticipants); err != nil {
+			ErrGenericInternalServerError.WithErr(err).Write(w)
+			return
+		}
+	}
+
+	// Add processes to the bundle
+	if err := a.db.AddProcessesToBundle(bundleID, processesToAdd); err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	httpWriteJSON(w, CreateProcessBundleResponse{
+		URI:  "/process/bundle/" + bundleIDStr,
+		Root: bundle.CensusRoot,
+	})
+}
+
+// processBundleInfoHandler retrieves process bundle information by ID.
+// Returns bundle details including the associated census, census root, organization address, and list of processes.
+func (a *API) processBundleInfoHandler(w http.ResponseWriter, r *http.Request) {
+	bundleIDStr := chi.URLParam(r, "bundleId")
+	if bundleIDStr == "" {
+		ErrMalformedURLParam.Withf("missing bundle ID").Write(w)
+		return
+	}
+
+	bundleID, err := primitive.ObjectIDFromHex(bundleIDStr)
+	if err != nil {
+		ErrMalformedURLParam.Withf("invalid bundle ID").Write(w)
+		return
+	}
+
+	bundle, err := a.db.ProcessBundle(bundleID)
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
```

Comment:
Same here, I think that is better to convert the string to `ObjectID` into the `ProcessBundle` method.